wiki/People/Community_Contributors/en.md

```diff
@@ -1,34 +1,34 @@
 Community Contributors
-========================
+======================
 
 ![Contributor Badge](Contributor_Badge.png "Contributor Badge")
 
-The **Community Contributors** are the people who have taken it upon themselves to provide service to the [osu!](/wiki/Glossary) community at great cost to themselves - tools and features that many of us take for granted today. These users are given a badge to denote that they did something notable for the entire community.
+The **Community Contributors** are the people who have taken it upon themselves to provide service to the [*osu!*](/wiki/Glossary/#osu) community at great cost to themselves - tools and features that many of us take for granted today. These users are given a badge to denote that they did something notable for the entire community.
 
-Not to be mistaken as [osu! Alumni](/wiki/People/osu!_Alumni) which are previous osu!team members ([GMT](/wiki/People/Global_Moderation_Team), [QAT](/wiki/People/Quality_Assurance_Team) etc) that retired, Community Contributors badge are given to public freelancers who had go beyond the call of duty to improve osu! and the community environment.
+Not to be mistaken as [osu! Alumni](/wiki/People/osu!_Alumni) which are previous osu!team members ([GMT](/wiki/People/Global_Moderation_Team), [QAT](/wiki/People/Quality_Assurance_Team), etc.) that retired, Community Contributors badge are given to public freelancers who had go beyond the call of duty to improve *osu!* and the community environment.
```

Comment:
`Community Contributors badge` -> `Community Contributor badges`
(the `s` is in the wrong spot)